src/features/homepage/Homepage.js

```diff
@@ -0,0 +1,84 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import {
+  BsGraphUp, BsFillBarChartLineFill, BsFillCalculatorFill,
+  BsFillCalendar2DateFill, BsFillPieChartFill, BsFillRssFill,
+  BsChatQuoteFill, BsCoin, BsCpuFill, BsGraphDown, BsTerminalFill,
+  BsMic, BsGear, BsChevronLeft,
+} from 'react-icons/bs';
+import {
+  RiAtLine, RiCloudLine, RiCodeBoxFill, RiBaseStationLine,
+  RiComputerLine, RiRadarFill, RiSave3Fill, RiWifiFill,
+} from 'react-icons/ri';
+import { selectAllQuotes, selectState, fetchQoutes } from './homepageSlice';
+import Card from './Card';
+import './css/Homepage.css';
+
+// import headerImg from '../../assets/images/homeHeader1.png';
+
```

Comment:
- [ ] Let's try not to have commented-on code in our codebase as they make our code dirty and not look professional. It's a good convention to not leave unused codes commented in our codebase. Please remove all commented codes from your codebase 👍🏽